debian/rules

```diff
@@ -5,16 +5,21 @@ ifeq ($(findstring parallel=,$(DEB_BUILD_OPTIONS)),)
 endif
 
 %:
-	dh $@ --buildsystem=cmake --parallel
-
-DEB_BUILD_GNU_TYPE  ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
-UBUNTU_CODENAME  := $(shell lsb_release -sc)
-UBUNTU_RELEASE  := $(shell lsb_release -sr)
+	dh $@ --with quilt --buildsystem=cmake --parallel
 
 extra_flags += \
--DCMAKE_C_COMPILER=gcc-8 \
--DCMAKE_CXX_COMPILER=g++-8 \
--DCMAKE_BUILD_TYPE="Release" \
+-DCMAKE_C_COMPILER=gcc-10 \
+-DCMAKE_CXX_COMPILER=g++-10 \
+-DCMAKE_BUILD_TYPE="RelWithDebInfo" \
```

Comment:
Please, leave the Release build type!
Package is considered as a production version, in which we are stripping everything that is not needed for a player.

For any other purposes, e.g. debugging - custom build should be made from sources manually.

Reply:
During packaging deb, binaries are stripped and debug symbols are put into openxray-dbgsym package(so if someone want to fill bug report, they can install it and provide full backtrace) so binary looks like this:
file /usr/bin/xr_3da 
/usr/bin/xr_3da: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=86dead74faf750d85704dc78a7193342d7542018, stripped
"stripped" part indicates that there are no debug info there.
Same for libraries:
file /usr/lib/x86_64-linux-gnu/xrEngine.so
/usr/lib/x86_64-linux-gnu/xrEngine.so: ELF 64-bit LSB shared object, x86-64, version 1 (GNU/Linux), dynamically linked, BuildID[sha1]=b0b4edc31bf3824a70322795a3098c107c881c76, stripped

I'm using RelWithDebInfo so that I get release build with debug info, that is put in separate package.

Reply:
Oh, so `openxray-dbgsym` gets created automatically?
This is awesome! I wanted to suggest the idea of a separate package with debug symbols instead of `RelWithDebInfo`, but it seems that this is what we actually need.